### ISSUE TITLE
Create a default settings.json file if does not exist

### ISSUE THREAD

**Issue by martindsouza** [fuzziebrain/suez#2]
When starting suez if `settings.json` isn't found in the `config` folder a skeleton file should be created. Something like:

```json
{
  "service": {
    "port": 3000
  },
  "apiTargets": [
    {
      "name": "CHANGEME",
      "proxyTarget": "https://CHANGE.ME/"
    }
  ]
}

```

**Comment by fuzziebrain** [comment]
Was this taken care of in your recent PR or is this new?

**Comment by martindsouza** [comment]
No. It's something I think should be done. You can probably create the above sample file as `settings.default.json` and then if `settings.json` isn't found it'll move copy the file over in `app.js`. I like doing this in my applications so developers can get up and running right away.

For Docker users this will be very useful since they won't be required to even download anything from SUEZ.

**Comment by martindsouza** [comment]
No. It's something I think should be done. You can probably create the above sample file as `settings.default.json` and then if `settings.json` isn't found it'll move copy the file over in `app.js`. I like doing this in my applications so developers can get up and running right away.

For Docker users this will be very useful since they won't be required to even download anything from SUEZ.

**Comment by fuzziebrain** [comment]
There is a sample file included. :-)

**Comment by martindsouza** [comment]
Please hold since #8 may make this a moot point as we could get rid of `settings.json`
